tools/gen_parser/skulpt_parser_genarator.py

```diff
@@ -312,8 +314,12 @@ def visit_Alt(self, node: Alt, is_loop: bool, is_gather: bool) -> None:
                     if is_gather:
                         assert len(self.local_variable_names) == 2
                         action = f"[{self.local_variable_names[0]}, ...{self.local_variable_names[1]}]"
+                    elif len(self.local_variable_names) == 1:
+                        # @TODO is this correct?
+                        action = self.local_variable_names[0]
                     else:
-                        action = f"{', '.join(self.local_variable_names)}"
+                        # @TODO is it ok to return an array here
```

Comment:
in the python version it's one if one variable and a tuple if more then one. This looks sensible to me. we'll see where the parser falls over 😄